bundles/org.openhab.binding.gardena/src/main/resources/OH-INF/thing/bridge.xml

```diff
@@ -8,6 +8,7 @@
 	<bridge-type id="account">
 		<label>Gardena Smart System Account</label>
 		<description>The Gardena smart system account</description>
+		<semantic-equipment-tag>NetworkAppliance</semantic-equipment-tag>
```

Comment:
Is it a hub/gateway in your home or a cloud account?

Reply:
It is a physical hub.